lib/vmpooler/api/v1.rb

```diff
@@ -747,6 +803,58 @@ def invalid_templates(payload)
 
       JSON.pretty_generate(result)
     end
+
+    post "#{api_prefix}/config/poolsize/?" do
+      content_type :json
+      result = { 'ok' => false }
+
+      need_token! if Vmpooler::API.settings.config[:auth]
+
+      payload = JSON.parse(request.body.read)
+
+      if payload
+        invalid = invalid_template_or_size(payload)
+        if invalid.empty?
+          result = update_pool_size(payload)
+        else
+          invalid.each do |bad_template|
+            metrics.increment("config.invalid.#{bad_template}")
+          end
+          status 404
```

Comment:
I would add the content of the `invalid` array you built
something like
```
 result[:message] = invalid.map { |i| "Invalid pool size change: #{i}" }.join(",")
```

Reply:
+1 for adding a message to the result.  Would it also be helpful to add some logging info and error statements besides the metrics?  Just thinking about possibly needing to go back through the logs to validate whether or not a request was successful.

Reply:
We will get log messages from the pool_manager interactions. We don't otherwise track success or failures of API operations, or log anything from the API currently.